src/CashuWallet.ts

```diff
@@ -1,39 +1,56 @@
+import {
+	blindMessage,
+	constructProofFromPromise,
+	serializeProof
+} from '@cashu/crypto/modules/client';
+import { deriveBlindingFactor, deriveSecret } from '@cashu/crypto/modules/client/NUT09';
```

Comment:
Can you please double check the imports? It seems like there is a bunch of duplication going on. For example is this line duplicated in line 61